bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java

```diff
@@ -129,25 +131,33 @@ public Transformer createTransformer() {
         return new VersionableClientlibsTransformer();
     }
 
-    private Attributes versionClientLibs(final String elementName, final Attributes attrs) {
+    private Attributes versionClientLibs(final String elementName, final Attributes attrs, String contextPath) {
```

Comment:
There are two methods with `String contextPath` argument but it is made `final` in only one of them.
Is it ok?